kousa/lib/kousa/room.ex

```diff
@@ -301,6 +301,44 @@ defmodule Kousa.Room do
         {:error, Kousa.Utils.Errors.changeset_to_first_err_message_with_field_name(x)}
     end
   end
+  
+  def send_waiting_room_casts(user_id, room) do
```

Comment:
why are you doing 2 casts here

Reply:
I send one to the host of the room to show the modal, and the other one goes to the user.